pkgs/development/tools/dstp/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGo117Module, fetchFromGitHub }:
+
+buildGo117Module rec {
```

Comment:
```suggestion
buildGoModule rec {
```

Or do we explicitly need 1.17?

Reply:
Only 1.17 is supported, once 1.17 becomes the default in nix, I'll update it